osidb/migrations/0076_auto_20230526_1959.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.2.19 on 2023-05-26 19:59
```

Comment:
Could you give the file a non-auto generated name? (something describing the change) It is handy when later navigating through the migrations when trying to find something.

Reply:
Before the final squash and rebase, I'll make sure to regenerate the migration(s) with a meaningful name.